src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,23 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        int vectorX = toX - robot.getX();
+        int vectorY = toY - robot.getY();
+        Direction vectorDirectionX = vectorX > 0 ? Direction.RIGHT : Direction.LEFT;
```

Comment:
```suggestion
        Direction vectorDirectionX = (toX - robot.getX()) > 0 ? Direction.RIGHT : Direction.LEFT;
```
variable `vectorX` and `vectorY`  are redundant